hook.mjs

```diff
@@ -4,7 +4,7 @@
 
 const specifiers = new Map()
 
-const EXTENSION_RE = /\.(js|mjs|cjs)$/
+const EXTENSION_RE = /\.(js|mjs|cjs|ts|mts|cts)$/
```

Comment:
Could you add a comment here indicating that this is temporary until we figure out a better way of supporting arbitrary extensions?